CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Requests with empty body and application/json Content-Type Header will now
   return 400 error instead of 500 error.
   [cyberark/conjur#1968](https://github.com/cyberark/conjur/issues/1968)
+- Conjur now properly sets the 'operation' field of a secret update audit message.Previously, the operation for this event was empty.
```

Comment:
i don't think that we need the second sentence but if you think so then please add a space after the dot. I would also move it to a new line.

and don't forget the link to the github issue.